WORKSPACE

```diff
@@ -2,13 +2,14 @@
 '''
 load("@bazel_tools//tools/build_defs/repo:http.bzl", "http_archive")
 load("@bazel_tools//tools/build_defs/repo:jvm.bzl", "jvm_maven_import_external")
+load("@bazel_tools//tools/build_defs/repo:git.bzl", "git_repository")
 
 # Android SDK configuration. For more details, see:
 # https://docs.bazel.build/versions/master/be/android.html#android_sdk_repository
 android_sdk_repository(
     name = "androidsdk",
     api_level = 28,
-    build_tools_version = "28.0.2",
+    # build_tools_version = "28.0.2",
```

Comment:
Prefer not commenting out code. Either remove it, or keep the actual code. Commented out code can become quickly out of date when checked in.

Reply:
Okay! Fixed.